qcodes/plots/pyqtgraph.py

```diff
@@ -99,6 +99,10 @@ def add_to_plot(self, subplot=1, **kwargs):
                 self.subplots.append(self.add_subplot())
         subplot_object = self.subplots[subplot - 1]
 
+        if 'name' in kwargs:
+            if subplot_object.legend is None:
+                subplot_object.addLegend(offset=(-30,30))
```

Comment:
actually I think it is in (screen) pixels... I just moved it from top left to top right.